app/controllers/api/design_files_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::DesignFilesController < ApplicationController
+  # PATCH/PUT /api/design_files/:designId
+  def update_last_modified
+    design_file = DesignFile.find(params[:designId])
```

Comment:
The update_last_modified method does not perform any form of authentication or authorization. This could allow an unauthenticated user to update the last_modified attribute of any DesignFile. Ensure that proper authentication and authorization checks are in place before allowing such updates.